ONSdigital/dp-frontend-renderer
# Add version label

### What

- Add version label to template
- Update vendored page model

### How to review

- Pull https://github.com/ONSdigital/dp-frontend-dataset-controller/pull/146
- View a legacy dataset that has versions that contain `description.versionLabel` value. It should appear next to the edition title 

### Who can review

Anyone
